inst/templates/head.html

```diff
@@ -4,7 +4,13 @@
 
 <title>{{{pagetitle}}} • {{#site}}{{title}}{{/site}}</title>
 
-<link rel="shortcut icon" type="image/x-icon" href="{{#site}}{{root}}{{/site}}favicon.ico">
+<link rel="icon" type="image/x-icon" href="{{#site}}{{root}}{{/site}}favicon.ico">
+<link rel="icon" type="image/png" sizes="16x16" href="{{#site}}{{root}}{{/site}}favicon-16x16.png">
+<link rel="icon" type="image/png" sizes="32x32" href="{{#site}}{{root}}{{/site}}favicon-32x32.png">
```

Comment:
I'm actually missing the `96x96` we talked about. This is used for android if users want to create an icon on their home screen. For that, we need to declare a manifest. Here the example given on the realfavicongenerator website:

```json
			"android_chrome": {
				"picture_aspect": "shadow",
				"manifest": {
					"name": "My sample app",
					"display": "standalone",
					"orientation": "portrait",
					"start_url": "/homepage.html",
					"existing_manifest": "{\"name\": \"Yet another app\"}"
				},
				"assets": {
					"legacy_icon": true,
					"low_resolution_icons": false
				},
				"theme_color": "#4972ab"
			}
```

Do you think it's useful to go this far?

Reply:
Probably not yet, lets leave it for now.